.github/workflows/python-dev.yml

```diff
@@ -40,12 +48,16 @@ jobs:
         python-version: '3.10-dev'
 
     - name: Install dependencies
+      shell: bash
       run: |
         python -m pip install --upgrade pip setuptools wheel
-        pip install -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
-        pip install git+https://github.com/pytest-dev/pytest.git
+        if [[ "$NUMPY_WHEELS_AVAILABLE" == "true" ]]; then
+          pip install -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
+        else
+          pip install git+https://github.com/numpy/numpy.git
+        fi
         pip install git+https://github.com/nedbat/coveragepy.git
-        pip install cython python-dateutil pytz hypothesis pytest-xdist pytest-cov
+        pip install cython python-dateutil pytz hypothesis pytest>=6.2.5 pytest-xdist pytest-cov
```

Comment:
is this in our requirements-dev?

Reply:
Its 6.0 in requirements dev. 6.2.5 is the first version to support python 3.10

Reply:
oh ok got it, can you update whatsnew with this as well